test/unit/org/apache/cassandra/security/DummySslContextFactoryImpl.java

```diff
@@ -61,6 +61,12 @@ public boolean shouldReload()
         return false;
     }
 
+    @Override
+    public boolean hasOutboundKeystore()
```

Comment:
Won't need this if we follow Jon's suggestion of having interface level default.